variables.tf

```diff
@@ -51,3 +51,16 @@ variable "task_role_arn" {
   default     = ""
   description = "The arn of the iam role you wish to pass to the ecs task containers."
 }
+
+variable "ordered_placement_strategies" {
+  type = list(object({ type = string, field = string }))
+  default = [{
```

Comment:
I know this is a nitpick, but I dislike this curly brace placement. I think it'd be easier to figure out which closing brace it corresponds to if you put it on its own line (same for line 60).